packages/shared/src/lib/flow-run/execution/step-output.ts

```diff
@@ -11,6 +14,7 @@ export class StepOutput<T = any>{
   errorMessage?: unknown;
   standardOutput?: unknown;
   status?: StepOutputStatus;
+  pauseMetadata?: Omit<PauseMetadata, 'executionState'>
```

Comment:
Why this should exists inside StepOutput? Maybe we can use the global in engine if we want to only the one that paused?

Reply:
because it needs to be recursive